src/geophires_x/Units.py

```diff
@@ -338,3 +340,13 @@ class HeatPerUnitVolumeUnit(str,Enum):
 class PowerPerUnitVolumeUnit(str,Enum):
     """Population Density Units"""
     MWPERCUBICKM = "MW/km**3"
+
+
+class Decay_RateUnit(str,Enum):
+    """Decay rate Units"""
+    PERCENTPERYEAR = "%/yr"
+
+
+class Inflation_RateUnit(str,Enum):
+    """Decay rate Units"""
+    KPASCALPERYEAR = "kPa/yr"
```

Comment:
In general I would avoid adding new enums here since they are probably redundant with passing around Pint quantities (although haven't had time in this moment to look at where these are used, so this is a general comment that may not be fully applicable in this context)

Reply:
I think we do need these new units, as they represent new values that some new Parameters are defining. I don't see how using Pint quantities gets us around defining new values when we need them. I am going to leave them.